topologies/datacenter-latest/labguides/source/bgp-with-cvp.rst

```diff
@@ -0,0 +1,170 @@
+Layer 3 Leaf-Spine
+==================
+
+.. image:: images/cvp-l3ls/l3ls_1.png
```

Comment:
This image can be removed, as I will be adding in the topology image with image map via the `labUI.py` script. Could you also add a note in here informing the user to click on a network device to ssh into that device?

Reply:
On second thought, you can keep the image here, the `labUI.py` script takes this image into account and ignores it. Just add in a Note about being able to access the nodes.